FrameworkSystem/Client/SystemAdministratorClientCLI.py

```diff
@@ -280,7 +280,13 @@ def do_show( self, args ):
         fields = ['Parameter','Value']
         records = []
         for key, value in result['Value'].items():
```

Comment:
Minor comment: I think that "key" and "value" are not descriptive names. At the same time you may use iteritems() instead of items() which gives better results.